checkstyle/spotbugs-exclude.xml

```diff
@@ -35,6 +35,14 @@
     <Match> <!-- Returning a java.lang.Boolean is common with Streams/Lambdas -->
         <Bug pattern="NP_BOOLEAN_RETURN_NULL" />
     </Match>
+    <Match> <!-- fromBytes was not reporting this warning in earlier version of checkstyle -->
```

Comment:
Please don't add these suppressions are these are almost certainly true positives. 
If it means changing some code to fix the issues that's fine.